posthog/hogql_queries/insights/trends/aggregation_operations.py

```diff
@@ -48,26 +50,120 @@ class AggregationOperations:
     series: EventsNode | ActionsNode
     query_date_range: QueryDateRange
 
-    def __init__(self, series: str, query_date_range: QueryDateRange) -> None:
+    def __init__(self, series: EventsNode | ActionsNode, query_date_range: QueryDateRange) -> None:
         self.series = series
         self.query_date_range = query_date_range
 
     def select_aggregation(self) -> ast.Expr:
-        if self.series.math == "hogql":
+        if self.series.math == "hogql" and self.series.math_hogql is not None:
             return parse_expr(self.series.math_hogql)
         elif self.series.math == "total":
             return parse_expr("count(e.uuid)")
         elif self.series.math == "dau":
             return parse_expr("count(DISTINCT e.person_id)")
         elif self.series.math == "weekly_active":
-            return ast.Field(chain=["counts"])
+            return ast.Field(chain=["counts"])  # This gets replaced when doing query orchestration
```

Comment:
"This gets replaced" sounds a lot like `ast.Placeholder` 😄 I'm curious why this scheme instead of placeholders?

Reply:
Either is fine - no AST parsing/resolving is actually run before the replacement gets done, we just need _something_ to populate the field. In my mind, `Field` is the simplest AST node we could pick here

Reply:
If they both function the same way here, I think it'd be better to use `Placeholder`, clearer in terms of intent. If that'd need more complex replacement logic, then `Field` is fine

Reply:
Updated